Template/LO/SubProcesses/idenparts.f

```diff
@@ -31,6 +31,7 @@ subroutine idenparts(iden_part,itree,sprop,forcebw,prwidth)
 c
 c     Global
 c
+      include 'vector.inc'
       include 'coupl.inc'                     !Mass and width info
```

Comment:
same here, why is this needed?

Reply:
same as above, vector.inc is needed by coupl.inc